2.13/tests/test/test.js

```diff
@@ -432,6 +432,42 @@ describe('DELETE /v2/service_instance/:instance_id', function() {
     });
 });
 
+function testInstanceLastOperationStatus(instance_id, done) {
+    endpoint = '/v2/service_instances/' + instance_id + '/last_operation'
+
+    testAPIVersionHeader(endpoint, 'GET');
+    testAuthentication(endpoint, 'GET');
+
+    var pollDuration = 0;
+    var succeededOrFailed = false;
+    var interval = setInterval(function() {
+        pollDuration += config.polling_interval;
+        if (succeededOrFailed || (pollDuration > config.max_polling_duration)) {
+            clearInterval(interval);
+            return done();

```

Comment:
Mocha has a convenient function [`this.timeout()`](https://mochajs.org/#timeouts) to restrict duration for test in different levels. So, `max_polling_duration` is a little more complicated here.